src/converters/degiroConverterV3.test.ts

```diff
@@ -35,7 +35,7 @@ describe("degiroConverterV3", () => {
       // Assert
       expect(actualExport).toBeTruthy();
       expect(actualExport.activities.length).toBeGreaterThan(0);
-      expect(actualExport.activities.length).toBe(26);
+      expect(actualExport.activities.length).toBe(43);
```

Comment:
I see the [tests are currently failing](https://github.com/dickwolff/Export-To-Ghostfolio/actions/runs/12699976144/job/35632606126). V2 converter is missing 1 (as V2 and V3 tests are sharing the same input file for testing), and V3 only got 31 instead of expecting the 43 you put in the test. Can you take a look at this and make changes?

Reply:
@dickwolff I need help with tests:
1. to update Yahoo Finance information. I think the instructions from the wiki are not complete. I got a missing import of a Mock object, which I didn't figure out how to import. Note that I'm a first-time user of Typescript
2. How do I print debug statements (to stdout/stderr) in tests? I'm getting unexpected behavior, which I need to debug.

Reply:
Sure! Hope this helps:

1. If you use an IDE like VSCode, you can to `CTRL + .` (dot) and then import all usings. 
2. In the tests, console statements are supressed. You can comment the whole `beforeEach(() => {` function, then you'll get the console log statements in your console. For debugging purposes I usually add `console.log(...)` statements on places that could help me debug.